packages/wrangler/src/cloudchamber/build.ts

```diff
@@ -78,20 +79,11 @@ export async function buildAndMaybePush(
 	pathToDocker: string,
 	push: boolean,
 	containerConfig?: ContainerApp,
+	accountId?: string,
```

Comment:
This shouldn't be optional

```suggestion
	accountId: string,
```